t/op/signatures.t

```diff
@@ -1608,6 +1613,82 @@ while(<$kh>) {
             'f($1)';
 }
 
+# check that various uses of @_ inside signatured subs causes "experimental"
+# warnings at compiletime
+{
+    sub warnings_from {
+        my ($code, $run) = @_;
```

Comment:
consider using signature? it's available here, no?
if so then same comment to all added helpers

Reply:
Ah - for entirely selfish implementation reasons I didn't want to use any signatured subs for the implementation of the unit-test which tests signatured subs. It means I can add all sorts of "print to STDERR" or other messy debugging trickery, and not get fake noise from testing.

Reply:
I understand the fear, I jus noticed that we were using it above in this file